pybliometrics/utils/get_content.py

```diff
@@ -99,19 +108,36 @@ def get_content(url, api, params=None, **kwds):
         except (IndexError, ValueError):
             pass
 
-    # Perform request, eventually replacing the current key
-    timeout = config.getint("Requests", "Timeout", fallback=20)
-    resp = session.get(url, headers=header, proxies=proxies, params=params,
-                       timeout=timeout)
+    # Either use token header or key header
+    if token_key:
+        header = token_header
```

Comment:
What is this good for?

Reply:
There is actually no need to define a new `header` variable. We can directly pass the `token_header` and `key_header` to the `session.get()`